geomesa-blobstore/geomesa-blobstore-core/src/main/scala/org/locationtech/geomesa/blob/core/handlers/BlobStoreFileHandler.scala

```diff
@@ -0,0 +1,56 @@
+package org.locationtech.geomesa.blob.core.handlers
+
+import java.io.File
+import java.util
+import java.util.Date
+import javax.imageio.spi.ServiceRegistry
+
+import com.vividsolutions.jts.geom.Geometry
+import org.geotools.feature.simple.SimpleFeatureBuilder
+import org.locationtech.geomesa.accumulo.util.{Z3FeatureIdGenerator, Z3UuidGenerator}
+import org.locationtech.geomesa.blob.core.AccumuloBlobStore._
+import org.opengis.feature.simple.SimpleFeature
+
+import scala.collection.JavaConversions._
+
+object BlobStoreFileHandler {
+  def buildSF(file: File, params: Map[String, String]): Option[SimpleFeature] = {
```

Comment:
seems like since this is a generic method is should go in a generic FileHandler companion object? Of course since FileHandler is in java maybe that's confusing or something.